mlir/utils/jenkins/Jenkinsfile

```diff
@@ -505,224 +542,217 @@ pipeline {
                 skipDefaultCheckout()
             }
             steps {
-                // Note: add additional architectures here
-                dir ('MIOpen/build/MIOpenUserDB') {
-                    unstash name: "MLIR-PerfDB-${params.canXdlops ? 'gfx908' : 'gfx906'}"
-                    // This'll stop being a duplicate once Navi CI starts
-                    unstash name: "MLIR-PerfDB-${params.canXdlops ? 'gfx90a' : 'gfx906'}"
-                    sh 'date --utc +%Y-%m-%d >tuning-date'
-                }
-                archiveArtifacts artifacts: 'MIOpen/build/MIOpenUserDB/**',\
-                    onlyIfSuccessful: true
+                archivePerfDB()
```

Comment:
SInce I can't attach to that line of code, I'll point out we don't have to say "xdlops" in the stage name here